driver-core/src/main/com/mongodb/client/model/MongoTimeUnit.java

```diff
@@ -68,7 +73,13 @@ public enum MongoTimeUnit {
         this.fixed = fixed;
     }
 
-    String value() {
+    /**
+     * Returns a {@link String} representation of the unit, which may be useful when using methods like
+     * {@link Windows#of(Bson)}, {@link DensifyRange#of(Bson)}.
+     *
+     * @return A {@link String} representation of the unit.
+     */
+    public String value() {
```

Comment:
Had to make it `public` to be able to access it from the new `com.mongodb.client.model.densify` package.

Reply:
Ack